junebug/stores.py

```diff
@@ -0,0 +1,53 @@
+from twisted.internet.defer import inlineCallbacks, returnValue
+from vumi.message import TransportUserMessage
+
+
+class BaseStore(object):
+    '''Base class for store objects. Stores data in redis in a hash.
+    redis: redis manager
+    ttl: expiry for keys in the store
```

Comment:
Does sphinx require this to be the class docstring or the `__init__` method docstring?